pywatts/modules/preprocessing/sampler.py

```diff
@@ -68,11 +68,24 @@ def transform(self, x: xr.DataArray) -> xr.DataArray:
         if not indexes:
             indexes = _get_time_indexes(x)
         try:
-            r = [x.shift({index: i for index in indexes}, fill_value=0) for i in range(0, self.sample_size)]
```

Comment:
I would suppose that issue 174 is a bug. Therefore, you do not implement the if else statement. Instead you can replace line 71 with: ` r = [x.shift({index: i for index in indexes}, fill_value=0) for i in range(self.sample_size - 1, -1, -1)]
`